home/models.py

```diff
@@ -90,11 +91,18 @@ def get_paragraph_values(self):
         paragraph_values = self._get_child_block_values('paragraph')
         return '\n'.join(paragraph_values)
 
+    def get_absolute_url(self):
+        return self.get_url()
+
     content_panels = Page.content_panels + [
         ImageChooserPanel('lead_image'),
         StreamFieldPanel('body')
     ]
 
+    settings_panels = Page.settings_panels + [
+        FieldPanel('allow_comments')
```

Comment:
My preference would be for the panel to be titled 'Comments', and for there to be a checkbox labelled 'Enabled' within the panel.